packages/leancode_markup/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## Unreleased
+## 0.1.0
 
+- Added `style` parameter to `MarkupText`.
```

Comment:
One last thing - we should write changelog in imperative verb form, same like in commits
```suggestion
- Add `style` parameter to `MarkupText`.
```